src/isomorphic/hooks/ReactComponentTreeHook.js

```diff
@@ -39,6 +43,24 @@ function isNative(fn) {
   }
 }
 
+type Item = {
+  element: ReactElement,
+  parentID: DebugID,
+  text: ?string,
+  childIDs: Array<DebugID>,
+  isMounted: bool,
+  updateCount: number,
+};
+
+declare function getItem(id: DebugID): ?Item;
```

Comment:
I'm doing two implementations in the two branches below, I want both to match the same API. What is the flow way of doing things?